### PR TITLE
Fix List wrapping bug

### PR DESCRIPTION
**Pull request by danielamorse**
## Jira

http://vjira2:8080/browse/BDS-2087

## Summary

Fixes Safari sub-pixel rendering bug.

## Details

Setting a width on List was required to fix a FF-only bug, but it turns out it causes a different bug in Safari. To fix Safari, simply scope the `width` rule to FF-only.

Supercedes #1805.

## How to test
Review the following pages in Chrome, Safari, FF, and Edge. Verify they work as expected:
- [Completed Training template](https://fix-list-wrapping-bug.boltdesignsystem.com/pattern-lab/patterns/03-blueprints-05-pages-completed-training-completed-training/03-blueprints-05-pages-completed-training-completed-training.html)
- [List demos](https://fix-list-wrapping-bug.boltdesignsystem.com/pattern-lab/patterns/02-components-list/index.html)

### CODE REVIEW

File: packages/components/bolt-list/src/list.scss [hunk 1]
```diff
@@ -68,28 +70,51 @@
...
     margin-bottom: bolt-v-spacing(#{$spacing-value-name}) * -1;
     margin-left: bolt-spacing(#{$spacing-value-name}) * -1;
+  }
 
-    // The inline here is talking about the items inside, the List component itself is still a block level element that would fill up the space of any container.
-    &.c-bolt-list--display-inline,
-    &.c-bolt-list--display-flex {
-      width: calc(100% + #{bolt-spacing($spacing-value-name)}); // Width must be defined in order for the list to dislay correctly in Firefox.
-    }
+  @-moz-document url-prefix('') {
```
Comment (sghoweri) [inline packages/components/bolt-list/src/list.scss:82]: Wait, what? Link to docs and/or comment about this please.

Comment (mikemai2awesome) [reply packages/components/bolt-list/src/list.scss:82]: Agree. Let's throw a comment in there. I've been using that, it's quite handy.

Comment (mikemai2awesome) [reply packages/components/bolt-list/src/list.scss:82]: https://css-tricks.com/snippets/css/css-hacks-targeting-firefox/
https://stackoverflow.com/questions/952861/targeting-only-firefox-with-css

Comment (danielamorse) [reply packages/components/bolt-list/src/list.scss:84]: @sghoweri added a comment, [see commit](https://github.com/boltdesignsystem/bolt/pull/1806/commits/71f771bd659faa24fc8d95c83dcf165b8514e385).

Comment (sghoweri) [reply packages/components/bolt-list/src/list.scss:84]: 👍

**Comment by sghoweri** [comment]
PR was released with v2.21.0

**Pull request merged**